packages/edit-site/src/components/sync-state-with-url/use-init-edited-entity-from-url.js

```diff
@@ -189,12 +195,25 @@ function useResolveEditedEntityAndContext( { postId, postType } ) {
 			return { postType, postId };
 		}
 
+		// Some URLs in list views are different
```

Comment:
I'm a bit confused with the logic of this hook. Specifically as to why sometimes we set the context with `postType`/`postId` and other times we set `postType`/`postId` directly. This is a bit outside the scope of this PR, so I won't block it – though I'd welcome some pointer for me to learn up (and possibly ways to improve in follow-ups).

I've tested that the Pages and Templates with `postId` (list view with item selected) work fine.

Reply:
So the return of this hook gives the postType and postId that is going to be rendered in the "preview" and the "editor". 

For "pages", that postType and postId is the "template" of the page, not the page directly because it's the root element. So in order to fill the template with the right postId, we provide it as "context" (BlockContext for the editor which ensures blocks like postContent block... will use the right postId).

This kind of relates with the unification efforts between post and site editors. I think ultimately, there are ways to simplify this further but there's some editSite state that forces us to keep this hook for now.

Reply:
That's helpful, thanks for the context.